src/vizier/services/metadata/metadata_server.go

```diff
@@ -73,13 +74,15 @@ func init() {
 	pflag.String("pod_namespace", "pl", "The namespace this pod runs in. Used for leader elections")
 	pflag.String("nats_url", "pl-nats", "The URL of NATS")
 	pflag.Bool("use_etcd_operator", false, "Whether the etcd operator should be used instead of the persistent version.")
+	pflag.StringSlice("metadata_namespaces", []string{v1.NamespaceAll}, "The list of namespaces to watch for metadata.")
 
 	// Metadata flags are set using the env vars in pl-cluster-config.
 	// We historically set PL_ETCD_OPERATOR_ENABLED but not PL_USE_ETCD_OPERATOR in the configmap.
 	// We also don't have a clean way to update  configmaps for existing deploys.
 	// So instead just map PL_ETCD_OPERATOR_ENABLED to use_etcd_operator to make it work.
 	// TODO: We should clean this up in the future and make these flags consistent.
 	viper.BindEnv("use_etcd_operator", "PL_ETCD_OPERATOR_ENABLED")
+	viper.BindEnv("metadata_namespaces")
```

Comment:
This isn't necessary. Our shared service setup code will bind all flags to corresponding env vars with a `PL_` prefix.